contrib/DEBIAN/control

```diff
@@ -0,0 +1,20 @@
+Source: zcash
+Section: utils
+Priority: optional
+Maintainer: Zcash Company <team@z.cash>
+Homepage: https://z.cash
+Build-Depends: autoconf, automake, bsdmainutils, build-essential
+ git, g++-multilib, libc6-dev, libtool
+ m4, ncurses-dev, pkg-config, python
+ unzip, wget, zlib1g-dev
+Vcs-Git: git@github.com:zcash/zcash.git
+Vcs-Browser: https://github.com/zcash/zcash
+Package: zcash
+Version: 1.0.0-rc1
+Architecture: amd64
+Description: Zcash is an implementation of the "Zerocash" protocol.
```

Comment:
@str4d This broke the dpkg-deb build, so I'm inclined to collapse the Source and Package back into one paragraph.... It might not follow the recommended policy, but it works.